README.md

```diff
@@ -6,6 +6,9 @@ Collaborators: James Feore
 ## Deployment:
 Deployed on Heroku at https://rocky-inlet-61764.herokuapp.com/
 
+## Test instructions:
+Test should run on their own as dummy environmental variables are included in tests.py.
+
 
 ## Tests:
 ### Test plan
```

Comment:
Consider putting your tox output in a code block (ie triple backtick), might make it easier on the eye.